Kucoin.Net/Interfaces/Clients/SpotApi/IKucoinClientSpotApiTrading.cs

```diff
@@ -137,7 +138,7 @@ public interface IKucoinClientSpotApiTrading
         /// <param name="pageSize">The amount of results per page</param>
         /// <param name="ct">Cancellation token</param>
         /// <returns>List of orders</returns>
-        Task<WebCallResult<KucoinPaginated<KucoinOrder>>> GetOrdersAsync(string? symbol = null, OrderSide? side = null, OrderType? type = null, DateTime? startTime = null, DateTime? endTime = null, OrderStatus? status = null, int? currentPage = null, int? pageSize = null, CancellationToken ct = default);
+        Task<WebCallResult<KucoinPaginated<KucoinOrder>>> GetOrdersAsync(TradeType tradeType = TradeType.SpotTrade, string? symbol = null, OrderSide? side = null, OrderType? type = null, DateTime? startTime = null, DateTime? endTime = null, OrderStatus? status = null, int? currentPage = null, int? pageSize = null, CancellationToken ct = default);
```

Comment:
Added TradeType parameter. Default value preserves previous behavior when calling with all parameters omitted, but all other calls will need to include TradeType.